www/data/docs_sidebar.yml

```diff
@@ -1,122 +1,195 @@
+        link: "#hab-origin-key-upload"
+      - title: hab pkg binlink
+        link: "#hab-pkg-binlink"
+      - title: hab pkg build
+        link: "#hab-pkg-build"
+      - title: hab pkg demote
+        link: "#hab-pkg-demote"
+      - title: hab pkg exec
+        link: "#hab-pkg-exec"
+      - title: hab pkg export
+        link: "#hab-pkg-export"
+      - title: hab pkg hash
+        link: "#hab-pkg-hash"
+      - title: hab pkg install
+        link: "#hab-pkg-install"
+      - title: hab pkg path
+        link: "#hab-pkg-path"
+      - title: hab pkg promote
+        link: "#hab-pkg-promote"
+      - title: hab pkg provides
+        link: "#hab-pkg-provides"
+      - title: hab pkg sign
+        link: "#hab-pkg-sign"
+      - title: hab pkg upload
+        link: "#hab-pkg-upload"
+      - title: hab pkg verify
+        link: "#hab-pkg-verify"
+      - title: hab plan init
+        link: "#hab-plan-init"
+      - title: hab ring key export
+        link: "#hab-ring-key-export"
+      - title: hab ring key generate
+        link: "#hab-ring-key-generate"
+      - title: hab ring key import
+        link: "#hab-ring-key-import"
+      - title: hab svc key generate
+        link: "#hab-service-key-generate"
+      - title: hab studio
+        link: "#hab-studio"
+      - title: hab sup
+        link: "#hab-sup"
+      - title: hab user key generate
+        link: "hab-user-key-generate"
+  - title: Best Practice Guides <span>&plus;</span>
+    link: "/docs/best-practices"
+    sub_links:
+    - title: Quickstart Guide
+      link: "#quickstart-guide"
+    - title: Container Orchestration
+      link: "#container-orchestration"
+    - title: Kubernetes
+      link: "#kubernetes"
+    - title: "ECS and Habitat"
+      link: "#ecs-and-habitat"
+    - title: "Apache Mesos and DC/OS"
+      link: "#mesos-dcos"
+    - title: Core Contribution
+      link: "#core-development"
+    - title: Advanced Plan.sh Writing
+      link: "#planwriting-guide"
+    - title: Iterative Development
+      link: "#iterative-development"
+    - title: Continuous Deployment
+      link: "#continuous-deployment"
+    - title: "Binary Wrapper Packages"
+      link: "#binary-wrapper"
+  - title: Reference <span>&plus;</span>
+    link: "/docs/reference"
+    sub_links:
+    - title: Environment Variables
+      link: "#environment-variables"
+    - title: Plan Settings
+      link: "#plan-settings"
+    - title: Plan Variables
+      link: "#plan-variables"
+    - title: Plan Helper Functions
+      link: "#utility-functions"
+    - title: Build Phase Callbacks
+      link: "#reference-callbacks"
+    - title: Application Lifecycle Hooks
+      link: "#reference-hooks"
+    - title: Handlebars Helpers
+      link: "#handlebars-helpers"
+    - title: Template Data
+      link: "#template-data"
+    - title: Package Contents
+      link: "#package-contents"
+    - title: Supervisor Log Keys
+      link: "#sup-log-keys"
+    - title: Infographics
```

Comment:
sidebar loses its fixed status and disappears when I click here

Reply:
The sidebar scrolls as you reach the bottom 10% of the page or something like that. As this is a long page and the infographics are the last article, when you click this link the sidebar is going to scroll upwards. Might be something we can do about it but for now its a relatively minor bug.